content/backend/graphql-js/1-getting-started.md

```diff
@@ -137,7 +137,7 @@ What you'll then see is a [GraphQL Playground](https://github.com/graphcool/grap
 
 ![](https://imgur.com/9RC6x9S.png)
 
-By clicking the green **SCHEMA**-button on the right, you can open the API documentation. This documentation is auto-generated based on your schema definition and displays all API operations and data types of your schema.
+By clicking the green **DOCS**-button on the right, you can open the API documentation. This documentation is auto-generated based on your schema definition and displays all API operations and data types of your schema.
```

Comment:
Hi! The `DOCS` button is no longer green. What do you think about keeping it just `clicking the  **DOCS** button`

Reply:
Updated it according to the comment.